app/main.py

```diff
@@ -1,52 +1,63 @@
 import json
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as ET # NOQA N817
 
 
-class Book:
-    def __init__(self, title: str, content: str):
+class ConsoleBook:
+    def display_console(self) -> None:
+        print(self.content)
+
+    def print_console(self) -> None:
+        print(f"Printing the book: {self.title}...")
+        print(self.content)
+
+
+class ReverseBook:
+    def print_reverse(self) -> None:
+        print(f"Printing the book in reverse: {self.title}...")
+        print(self.content[::-1])
+
+    def display_reverse(self) -> None:
+        print(self.content[::-1])
+
+
+class SerializeBook:
+    def serialize_json(self) -> str:
+        return json.dumps({"title": self.title, "content": self.content})
+
+    def serialize_xml(self) -> str:
+        root = ET.Element("book")
+        title = ET.SubElement(root, "title")
+        title.text = self.title
+        content = ET.SubElement(root, "content")
+        content.text = self.content
+        return ET.tostring(root, encoding="unicode")
+
```

Comment:
this isn't a good approach because more classes you have, more methods' names you should remember. It's better to create a common base class for each entity and inherit from it so that each child class should have an implementation of some method